src/handlers/check_commits/behind_master.rs

```diff
@@ -0,0 +1,58 @@
+use crate::github::{GithubClient, IssuesEvent};
+use tracing as log;
+
+/// Default threshold for parent commit age in days to trigger a warning
+pub const DEFAULT_PARENT_AGE_THRESHOLD: usize = 14;
+
+/// Check if the PR is behind the main branch by a significant number of commits
+/// or based on an old parent commit
+pub async fn behind_master(
+    age_threshold: usize,
+    event: &IssuesEvent,
+    client: &GithubClient,
+) -> Option<String> {
+    if !event.issue.is_pr() {
+        return None;
+    }
+
+    log::debug!("Checking if PR #{} is behind master", event.issue.number);
+
+    // First try the parent commit age check as it's more accurate
+    match event
+        .issue
+        .is_parent_commit_too_old(client, age_threshold)
+        .await
+    {
+        Ok(Some(days_old)) => {
+            log::info!(
+                "PR #{} has a parent commit that is {} days old",
+                event.issue.number,
+                days_old
+            );
+
+            return Some(format!(
+                "This PR is based on a commit that is {} days old. \
```

Comment:
```suggestion
                r"This PR is based on a commit that is {} days old.
```
You can use a multi-line string to avoid the backslashes.